go.mod

```diff
@@ -14,14 +14,14 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.11.3
 	github.com/hashicorp/golang-lru v0.5.5-0.20210104140557-80c98217689d
 	github.com/ipfs/go-log/v2 v2.5.1
-	github.com/libp2p/go-libp2p v0.22.0
+	github.com/libp2p/go-libp2p v0.23.0
 	github.com/libp2p/go-libp2p-pubsub v0.8.1
 	github.com/libp2p/go-libp2p-testing v0.12.0
 	github.com/minio/sha256-simd v1.0.0
 	github.com/mitchellh/mapstructure v1.5.0
 	github.com/multiformats/go-multiaddr v0.7.0
 	github.com/natefinch/atomic v1.0.1
-	github.com/oasisprotocol/curve25519-voi v0.0.0-20220708102147-0a8a51822cae
+	github.com/oasisprotocol/curve25519-voi v0.0.0-20220914075539-c3db9d0435d1
```

Comment:
separate PR for go.mod/go.sum changes? i don't see this PR needs this update.

Reply:
The PR can be found here: https://github.com/spacemeshos/go-spacemesh/pull/3595